Kernel/Graphics/GenericFramebufferDevice.cpp

```diff
@@ -33,7 +33,10 @@ ErrorOr<void> GenericFramebufferDevice::verify_head_index(int head_index) const
 
 ErrorOr<void> GenericFramebufferDevice::ioctl(OpenFileDescription&, unsigned request, Userspace<void*> arg)
 {
-    TRY(Process::current().require_promise(Pledge::video));
+    if (request != FB_IOCTL_GET_HEAD_EDID) {
+        // Allow anyone to query the EDID
```

Comment:
But why?

Reply:
This allows `DisplaySettings` to query this without having to pledge `"video"`. I feel like anyone should be able to do this (after all we should eventually expose this in `/sys` just like on e.g. Linux). However, if you disagree, there's some weird problem with `ConfigServer` when adding `"video"` to the pledges where it blows up saying `DisplaySettings` is not allowed to access its settings. I don't know what the deal is, but again it feels like it should be accessible to anyone.

Reply:
I suppose I can clarify this a bit more in the comment